packages/backend/src/common/types/email/email-options.ts

```diff
@@ -15,15 +17,10 @@ export interface EmailOptions {
 }
 
 export const defaultEmailOptions: EmailOptions = {
-  to: [
-    {
-      email: '',
-      name: '',
-    },
-  ],
+  to: [],
   from: {
-    email: '',
-    name: '',
+    email: getEnv('APP_EMAIL_FROM'),
```

Comment:
.env.example

We already have email sender data `packages/backend/src/common/enums/services/email/index.ts` but it would be great to move it to .env

Does env variables handled? Try to use `APP_EMAIL_FROM=vsebude.team@gmail.com` and `APP_NAME="Vse Bude"`